packages/backend/src/routes.ts

```diff
@@ -34,6 +49,57 @@ mainRouter.post(
   }
 )
 
+// asset
+mainRouter.post(
+  '/assets',
+  passport.authenticate('jwt', { session: false }),
+  createAsset
+)
```

Comment:
In what scenarios we will have to hit this endpoint?

Reply:
it is more like an admin endpoint, so you can add some assets for the whole app
maybe make sense to not have it and just use hit rafiki directly for that?

Reply:
> maybe make sense to not have it

I think so. Since the first scaffold for Rafiki Admin is almost done,(this includes the possibility to create an asset), we should be able create assets directly from there or just seed the database when the docker container spins up.